data/mods/ssb/random-teams.ts

```diff
@@ -412,6 +412,12 @@ export const ssbSets: SSBSets = {
 		signatureMove: 'OwO wuts dis?',
 		evs: {atk: 252, spd: 4, spe: 252}, nature: 'Adamant',
 	},
+	Naziel: {
+		species: "Kirlia", ability: 'Prankster', item: 'Eviolite', gender: ['M','F'],
```

Comment:
```suggestion
		species: "Kirlia", ability: 'Prankster', item: 'Eviolite', gender: '',
```

Reply:
I don't think arrays of gender are supported. If it's `''`, it'll randomly pick M/F/N.

Reply:
Nah empty strings are m/f. No property is just m/f/n